test/Microsoft.TemplateEngine.Core.UnitTests/StubSettingsLoader.cs

```diff
@@ -0,0 +1,81 @@
+﻿using Microsoft.TemplateEngine.Abstractions;
+using System;
+using System.Collections.Generic;
+using Microsoft.TemplateEngine.Abstractions.Mount;
+
+namespace Microsoft.TemplateEngine.Core.UnitTests
+{
+    internal class StubSettingsLoader : ISettingsLoader
```

Comment:
I only see two implementations of the ISettingsLoader interface the real implementation and this stub I added.